etc/picongpu/taurus-tud/V100_picongpu.profile.example

```diff
@@ -76,7 +76,34 @@ export CXXFLAGS="-Dlinux"
 export TBG_SUBMIT="sbatch"
 export TBG_TPLFILE="etc/picongpu/taurus-tud/V100.tpl"
 
-alias getNode='srun -p ml --gres=gpu:6 -n 1 --mem=0 --cpus-per-task=44 --pty -t 2:00:00 bash'
+# allocate an interactive shell for two hours
+#   getNode 2  # allocates 2 interactive nodes (default: 1)
+function getNode() {
+    if [ -z "$1" ] ; then
+        numNodes=1
+    else
+        numNodes=$1
+    fi
+    export OMP_NUM_THREADS=7
+    srun --time=2:00:00 --nodes=$numNodes --ntasks=$((6 * $numNodes)) --ntasks-per-node=7 --cpus-per-task=7 --mem=0 --exclusive --gres=gpu:6 -p ml --pty bash
+}
+
+# allocate an interactive shell for two hours
+#   getDevice 2  # allocates 2 interactive devices on one node (default: 1)
+function getDevice() {
+    if [ -z "$1" ] ; then
+        numDevices=1
+    else
+        if [ "$1" -gt 6 ] ; then
+            echo "The maximal number of devices per node is 6." 1>&2
+            return 1
+        else
+            numDevices=$1
+        fi
+    fi
+    export OMP_NUM_THREADS=7
```

Comment:
As above: Why `OMP_NUM_THREADS=7`?

Reply:
This is on OpenMP specification that sets the number of span-able threads of each MPI rank to 7 (see [here](https://gcc.gnu.org/onlinedocs/libgomp/omp_005fget_005fnum_005fthreads.html) - which agrees with the fact that there are 7 cores per MPI rank available (plus a few extra)).